packages/material-ui-lab/src/internal/pickers/PickersToolbar.tsx

```diff
@@ -73,15 +78,18 @@ const PickerToolbar: React.FC<PickersToolbarProps & WithStyles<typeof styles>> =
     viewType = 'calendar',
   } = props;
 
+  const styleProps = { ...props };
```

Comment:
```suggestion
  const styleProps = props;
```

This should be safe, if this is internal component.